recipes/emacs/build.sh

```diff
@@ -0,0 +1,7 @@
+if [ "$(uname)" == "Darwin" ]; then
+    ./configure  --prefix=$PREFIX --without-x
+else
+    ./configure  --prefix=$PREFIX --x-includes=$PREFIX/include --x-libraries=$PREFIX/lib
+fi
+
+make -j4  && make -j4  install
```

Comment:
So should I just build non-parallel for now? Do the CI VMs even have multiple CPUs?